### PR TITLE
fix: rename cloud tls var 

### PR DESCRIPTION
**Pull request by ypoplavs**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by nicufk — APPROVED** [review]

File: charts/testkube-api/templates/deployment.yaml [hunk 1]
```diff
@@ -310,7 +310,7 @@ spec:
               value: "{{ .Values.cloud.uiUrl }}"
             {{- end}}
             {{- if not .Values.cloud.tls.enabled }}
-            - name: TESTKUBE_PRO_TLS_INSECURE
+            - name: TESTKUBE_CLOUD_TLS_INSECURE
```
Comment (rangoo94) [inline charts/testkube-api/templates/deployment.yaml:313]: Should it be renamed, considering that we have renamed all the variables to `TESTKUBE_PRO`?

Zrzut ekranu 2024-02-23 o 16 52 04
Zrzut ekranu 2024-02-23 o 16 52 22
Zrzut ekranu 2024-02-23 o 16 52 50

We are specifically using this one now.

Zrzut ekranu 2024-02-23 o 16 53 02

**Review by rangoo94 — CHANGES_REQUESTED** [review]

File: charts/testkube-api/templates/deployment.yaml (hunk 1 above)
Comment (ypoplavs) [reply charts/testkube-api/templates/deployment.yaml:313]: hm, I didn't see `TESTKUBE_PRO_TLS_INSECURE ` anywhere in testkube-cloud-api repo. At least, it didn't work when I used it in a chart.

Comment (rangoo94) [reply charts/testkube-api/templates/deployment.yaml:313]: It's not in `testkube-cloud-api` repo - it's in `testkube` repository.

Comment (ypoplavs) [reply charts/testkube-api/templates/deployment.yaml:313]: nvm, I used an old agent, closing this

**Pull request closed without merging**